### PR TITLE
Use soft delete

### PR DESCRIPTION
**Pull request by texasRanger09**

### CODE REVIEW

File: iOS/SkyService/Data/DataService/DataService+CartLineItems.swift [hunk 1]
```diff
@@ -12,9 +12,9 @@ extension DataService {
             .flatMapLatest { [weak self] (workspaceId) -> Observable<[CartLineItem]> in
                 guard let `self` = self else { return Observable.empty() }
 
-                var query = "workspaceId == '\(workspaceId)' && userId == '\(userId)' && orderId == null"
+                var query = "workspaceId == '\(workspaceId)' && userId == '\(userId)' && orderId == null && deleted == false"
                 if let orderId = orderId {
-                    query = "workspaceId == '\(workspaceId)' && userId == '\(userId)' && orderId == '\(orderId)'"
+                    query = "workspaceId == '\(workspaceId)' && userId == '\(userId)' && orderId == '\(orderId)' deleted == false"
```
Comment (okdistribute) [inline iOS/SkyService/Data/DataService/DataService+CartLineItems.swift:17]: I think you're missing an && here

**Review by okdistribute — CHANGES_REQUESTED** [review]

**Review by kndoshn — APPROVED** [review]
LGTM, thanks!

**Comment by magicwave** [comment]
After downloading/installing certificates from the Apple dev portal and futzing with automatically/not-automatically manage signing with Xcode, choosing Team DittoLive Incorporated for signing on SkyService Pax and a cocoapod, Xcode build still fails because of missing Env.swift file.

**Comment by magicwave** [comment]
> After downloading/installing certificates from the Apple dev portal and futzing with automatically/not-automatically manage signing with Xcode, choosing Team DittoLive Incorporated for signing on SkyService Pax and a cocoapod, Xcode build still fails because of missing Env.swift file.

Ok. I figured out that the Env.swift file needs to be built with the shell script, however build still fails because the resulting Env.swift defines only an empty struct. I suppose in order to build I'll need to create an app on the portal. Something in the README  would be helpful.

**Comment by okdistribute** [comment]
Hi @magicwave, I agree this should be included in the README, as part of "Getting Started.

I just invited you to the skyservice portal app so you can see the app_id and token.

**Comment by okdistribute** [comment]
@texasRanger09 Will this need the Android side as well?

**Comment by texasRanger09** [comment]
> @texasRanger09 Will this PR need to change the queries for the Android side as well?

Yes. I’m working on the android side now and will commit the changes in this PR

**Comment by magicwave** [comment]
> Hi @magicwave, I agree this should be included in the README, as part of "Getting Started.
> 
> I just invited you to the skyservice portal app so you can see the app_id.

Thanks Rae, it's progress. However, the portal `skyservice-prod` app is configured with an auth provider name, but in the SkyServices code it appears that the `DittoAuthenticationDelegate` expects Env.swift to be pre-configured with a static `DITTO_AUTH_PASSWORD` value for Ditto initialization. I'm not sure where that would come from.

**Comment by okdistribute** [comment]
I DMed you the password!

**Comment by kndoshn** [comment]
I'll work on README ASAP. Sorry for the inconvenience.

@okdistribute 
> Makes me think that we should have a private repository with the official secrets and necessary info to publish the demo apps.

For publishing apps, I think we should just put env vars on CI/CD.

For development, I'm not very sure where we should put the env vars. We could use 1password but easy to forget updating.

**Review by kndoshn — APPROVED** [review]

**Comment by kndoshn** [comment]
@okdistribute Can you approve and merge?

**Review by okdistribute — APPROVED** [review]

**Pull request merged**